### PR TITLE
[hold] add Scroll to noon on open action to date selector [LEI-277]

### PR DESCRIPTION
**Pull request by hmoco**

### CODE REVIEW

File: exp-player/addon/components/exp-free-response/component.js [hunk 1]
```diff
@@ -132,6 +132,15 @@ export default ExpFrameBaseComponent.extend(Validations, {
         }
     },
     actions: {
+        onOpen(obj) {
+            this.obj = obj;
+            Ember.run.later(this, function() {
+                let itemList = document.getElementById('ember-power-select-options-' + this.obj.uniqueId);
```
Comment (abought) [inline exp-player/addon/components/exp-free-response/component.js]: This searches the entire document; you can use `this.$()` to get a selector relative to only this component.

**Comment by abought** [comment]
Three quick notes:

1. Please add a PR description with relevant information before this can be considered for merge.
2. Is this a possible race condition? If so, what kind of cross-browser testing have you done? 
3. When we've resolved any questions, this PR has developed merge conflicts that will need to be addressed before merge.

**Comment by samanehsan** [comment]
@hmoco, can you please take a look at this PR when you get a chance? 🍂

**Comment by abought** [comment]
We haven't seen much activity on this in a while, so closing for now to focus on existing QA items. We can revisit if the client renews the request.

**Pull request closed without merging**